test/merge_test.js

```diff
@@ -0,0 +1,270 @@
+goog.require('helper');
```

Comment:
Working with KJ on Karma support, I was under the impression that we needed to have all tests declare a goog.module as well for things to work.  Or did we find a way to make that not necessary?

Reply:
KJ's pull for karma support is only for supporting older browsers to run tests with. Right now, the tests run on chrome so I am leaving it as is. I was thinking to go back later and change the setup of tests once that gets merged, instead of making all these test PRs dependent on PR #26 and messing up the changelog

Reply:
Ok KJ pinged me and the plan is to just push tests with updated syntax even though it may temporarily break the tests